8.x/theme-development/directories.md

```diff
@@ -1,3 +1,131 @@
 ## Directories
 
-[[toc]]
\ No newline at end of file
+[[toc]]
+
+There are many different parts that come together to make the Larammerce platform work. 
+In fact, in the base structure Larammerce, To maintain the class structure of directories in a database, you use the parent method of a tree pointer, a tree in which each object points to a parent. 
+
+Let's take a look at directory properties and relationships.
```

Comment:
`Directory` is a Laravel model that extends Eloquent BaseModel to manage the content of directories existing in the system. Let's take a shallow look at the properties and relationships of the Directory class.